### ISSUE TITLE
Attaching template to token no longer works

### ISSUE THREAD

**Issue by vScourge** [KayelGee/token-attacher#82]
My macros to attach templates to tokens are throwing errors now. The TA included example macro "Create Template and Attach to token" also has the same errors.

```
token-attacher.js:1096 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading '_id')
[Detected 1 package: token-attacher]
    at Object.attachElementsToToken (token-attacher.js:1096)
    at eval (eval at _executeScript (foundry.js:19933), <anonymous>:14:19)
```
This is with Foundry VTT Version 10 - Build 286 Stable, and Token Attacher 4.5.2

**Comment by KayelGee** [comment]
I've missed that one. 
Sadly I'll have to take a deeper look as it's actually not only the attaching that seems to be broken but the movement aswell.

**Comment by KayelGee** [comment]
Fixed in 4.5.5
